Rakefile

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env rake
+
+begin
```

Comment:
Is it possible to move all of the files related to testing into the `spec` directory? 
* Gemfile
* Rakefile
* accept.sh

If not, maybe a new top-level directory (`test`? `verify`? other?) that would contain the above files in addition to `spec`?

The idea is to collect all of the files related to this work into a single location, as opposed to sprinkling them in the top-level directory.